src/TiciTacaToeyGameEngine.ts

```diff
@@ -258,20 +262,19 @@ class TiciTacaToeyGameEngine implements GameEngine {
             "",
             message.connection
           );
-          this.games[gameId].timers[message.playerId] = new Timer(
-            timePerPlayer
-          );
         }
         const updatedPlayersList = uniq([
           ...this.games[gameId].players,
           message.playerId,
         ]);
-
+        const timePerPlayer = this.games[gameId].timePerPlayer;
         const gameReadyToStart =
           updatedPlayersList.length === this.games[gameId].playerCount;
 
         if (gameReadyToStart) {
```

Comment:
This if condition should be the last line in this switch case

Reply:
did that already, I'll push the changes once I can get TIME_OUT error notification to working